grpc/src/main/java/io/stargate/grpc/codec/VarintCodec.java

```diff
@@ -36,7 +36,8 @@ public ByteBuffer encode(@NonNull Value value, @NonNull ColumnType type) {
   @Override
   public Value decode(@NonNull ByteBuffer bytes, @NonNull ColumnType type) {
     if (bytes.remaining() == 0) {
-      return null;
```

Comment:
This could not have worked: protobuf/grpc code will throw NPE upon getting `null`. Null handling is done by caller, before this method gets called.